sshuttle/client.py

```diff
@@ -665,6 +665,25 @@ def dns_done(chan, data, method, sock, srcip, dstip, mux):
     debug3('dns_done: channel=%d src=%r dst=%r\n' % (chan, srcip, dstip))
     response = DNSRecord.parse(data)
     debug3('For the DNS request: %r   >>>>> DNS response: %r <<<<<<' % (dnsreqs2[chan], response))
+
+    if hasattr(response, 'header') and hasattr(response.header, 'rcode') and response.header.rcode != getattr(RCODE, 'NOERROR'):
+        if DNS_PROXY_SUFFIX1 != '':
+            question = dnsreqs2[chan].get_q()
+            qn = str(question.qname)
+
+            local_domain_to_ignore = '.'.join(DNS_PROXY_SUFFIX1.split('.')[-2:])
+            failed_requests_to_ignore = ['arpa.', local_domain_to_ignore + '.', local_domain_to_ignore]
```

Comment:
Since DNS_PROXY_SUFFIX1 is static, I would make the failed_requests_to_ignore static also.  Save a few cycle.